technology-overview.md

```diff
@@ -0,0 +1,84 @@
+# Technology overview
+
+[Agoric](https://agoric.com/about/) aims at enabling safe cooperation among mutually suspicious parties
+
+The way it does it is by making it easy to write secure smart contracts and making it possible to run these contracts on one or several blockchains. Agoric strategy to writing smart contracts is to write them in JavaScript using a framework called ERTP (Electronic Rights Transfer Protocol)
+
+
+## High-level view
+
+### JavaScript
+
+Agoric works to make [ECMAScript](https://www.ecma-international.org/publications/standards/Ecma-262.htm) (standard defining JavaScript) a suitable language for easily expressing secure smart contracts. Agoric works with ECMA TC39 (which includes browser makers and Node.js representatives) in that direction
+
+Agoric also works with [ECMA TC53](https://www.ecma-international.org/memento/tc53.htm) to make [SES](https://github.com/Agoric/SES) the JavaScript for embedded devices
+
+
+#### Long term
+
+- [Realms](https://github.com/tc39/proposal-realms)
+- [Eventual-send operator](https://github.com/Agoric/proposal-infix-bang)
+- [WeakRefs](https://github.com/tc39/proposal-weakrefs/)
+- Jessie, a subset of JavaScript that simplifies writing of deterministic smart contracts
+    - eslint-config-jessie (v1+)
+- SES (v1+), a secure JavaScript runtime to run untrusted code
+
+
+#### Short term
+
+- [Realms shim](https://github.com/Agoric/realms-shim)
+- Tooling
+        - [Safe modules](https://github.com/Agoric/safe-modules)
+            - [TOFU (Trust On First Use)](https://github.com/bmeck/tofu/)
+- [Jessie prototype](https://github.com/Agoric/Jessie), a subset of JavaScript that simplifies writing of deterministic smart contracts
+    - [eslint-config-jessie (v0.x)](https://github.com/Agoric/eslint-config-jessie) is an [ESLint](https://eslint.org/) plugin that enforces that some JavaScript files are valid Jessie
+- [SES prototype (v0.x)](https://github.com/Agoric/SES), a secure JavaScript runtime to run untrusted code
+
+
+### Smart Contracts in JavaScript
+
+Agoric works on [ERTP](https://github.com/Agoric/ERTP) (Electronic Rights Transfer Protocol), a framework to express secure smart contracts
+
+ERTP is to writing smart contracts in JavaScript what React is to write user interfaces in plain DOM
```

Comment:
Also give a server-side example so people don't think we're talking about user interfaces specifically.